Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,167 @@
-                    android:layout_marginEnd="@dimen/grid_1"
-                    app:layout_constraintEnd_toStartOf="@+id/sender_profile_image_view"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/sender_text_view"
-                    android:text="@{@string/email_recipient_to(email.recipientsPreview)}"
-                    android:textAppearance="?attr/textAppearanceCaption"
-                    android:textColor="@color/color_on_surface_emphasis_medium" />
-
-                <ImageView
-                    android:id="@+id/sender_profile_image_view"
-                    android:layout_width="@dimen/email_sender_profile_image_size"
-                    android:layout_height="@dimen/email_sender_profile_image_size"
-                    android:contentDescription="@string/email_sender_profile_content_desc"
-                    android:scaleType="centerCrop"
-                    app:glideCircularCrop="@{true}"
-                    app:glideSrc="@{email.sender.avatar}"
-                    app:layout_constraintTop_toTopOf="@id/sender_text_view"
-                    app:layout_constraintBottom_toBottomOf="@+id/recipient_text_view"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    tools:src="@drawable/avatar_3" />
-
-                <TextView
-                    android:id="@+id/body_text_view"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/grid_3"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/sender_profile_image_view"
-                    android:text="@{email.body}"
-                    android:textAppearance="?attr/textAppearanceBody1" />
-
-                <androidx.recyclerview.widget.RecyclerView
-                    android:id="@+id/attachment_recycler_view"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/grid_3"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/body_text_view"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    android:overScrollMode="never" />
-
-            </androidx.constraintlayout.widget.ConstraintLayout>
-
-        </com.google.android.material.card.MaterialCardView>
-
+            android:paddingTop="@dimen/grid_3"
+            android:paddingBottom="@dimen/grid_3"
+            android:paddingHorizontal="@dimen/grid_2"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/email_app_bar_layout">
```

Comment:
You can delete these since MaterialCardView (this View's parent) is not a ConstraintLayout